src/driver/main.cpp

```diff
@@ -423,6 +424,15 @@ struct verify : command<verify>
     }
 };
 
+struct version : command<version>
+{
+    void parse(argument_parser) {}
```

Comment:
This should probably be `void parse(const argument_parser&) {}` to avoid the tidy warning.